content/projects/gsoc/2022/projects/automatic-git-cache-maintenance.adoc

```diff
@@ -14,30 +14,43 @@ mentors:
 - "rishabhbudhouliya"
 - "markewaite"
 links:
-  gitter: "jenkinsci/git-plugin"
-  draft: https://docs.google.com/document/d/1dHFEHbVPNyGRGtMF30df4O0UXoqgmEJpVVNoxxXYT1w/edit?usp=sharing
+  gitter: https://gitter.im/jenkinsci/git-plugin
```

Comment:
"gitter" is a jenkins.io macro: it appends automatically the `https://gitter.im/`.